src/main/java/uk/gov/hmcts/reform/judicialapi/controller/request/TestUserRequest.java

```diff
@@ -22,8 +22,8 @@
     private String forename = null;
     @JsonProperty("surname")
     private String surname = null;
-    @JsonProperty("password")
-    private String password = null;
+    @JsonProperty("pass")
```

Comment:
I think you need to have it as "password" as this is required for idam request

Reply:
modified code and working